### PR TITLE
Only load userzoom for non members

### PR DESCRIPTION
**Pull request by davidrapson**
This PR moves userzoom code to our JavaScript modules so we can control how it loads.

In this case, for the new survey we only want to load userzoom for non-members as the current survey doesn't apply to non-members.

screen shot 2015-03-25 at 17 33 56

### CODE REVIEW

File: frontend/assets/javascripts/src/utils/user.js [hunk 1]
```diff
@@ -82,8 +82,13 @@ define([
         };
     })();
 
+    var hasTier = function(memberDetail) {
+        return !!(memberDetail && memberDetail.tier);
+    };
```
Comment (feedmypixel) [inline frontend/assets/javascripts/src/utils/user.js]: this can be used else where in another PR as it is used in other areas

**Comment by feedmypixel** [comment]
:+1:

File: frontend/assets/javascripts/src/utils/user.js (hunk 1 above)
Comment (davidrapson) [inline frontend/assets/javascripts/src/utils/user.js]: Yep, that was my thinking behind adding it, little more concise than doing this same check everywhere.
